onebusaway-android/src/main/AndroidManifest.xml

```diff
@@ -145,7 +149,7 @@
                     android:name="android.support.PARENT_ACTIVITY"
                     android:value="org.onebusaway.android.ui.HomeActivity"/>
         </activity>
-        <activity android:name="org.onebusaway.android.ui.ReportStopProblemActivity"
+        <activity android:name=".report.ui.ReportStopProblemActivity"
```

Comment:
Let's keep the full package name here, just for consistency with the other activities.